### PR TITLE
WIP: please_cli: Added enable "option" to be able to disable projects (#1513)

### PR DESCRIPTION
**Pull request by rv404674**
add "enable" option to `please_cli/config.py` to be able to disable projects.

### CODE REVIEW

**Comment by garbas** [comment]
@rv404674 you only added the "enable" option. now you need to change `lib/please_cli/please_cli/decision_task.py` to take it into the account when it is generating deployment tasks.

**Comment by rv404674** [comment]
hey @garbas , in  'lib/please_cli/please_cli/decision_task.py, in order to take in account new enable parameter of 'config.py' i need to edit "https://github.com/mozilla/release-services/blob/master/lib/please_cli/please_cli/decision_task.py#L515",no??. I mean return  enable parameter from get_deploy_task() and check it here in line 515.

**Comment by garbas** [comment]
@rv404674 you found the right line where the change needs to happen. the "enable" option is already in `deploy_options` variable so you can just do:

```python
for index, (project, project_requires, deploy_target, deploy_options) in \
    enumerate(sorted(projects_to_deploy, key=lambda x: x[0])):
    if not deploy_options.get('enable'):
        continue
    project_uuid = slugid.nice().decode('utf-8')
    # ... the rest of the code from https://github.com/mozilla/release-services/blob/master/lib/please_cli/please_cli/decision_task.py#L515
    
```

**Comment by rv404674** [comment]
hey @garbas , Done. :). Please do take a look when you get some time. Thanks.

File: lib/please_cli/please_cli/decision_task.py [hunk 1]
```diff
@@ -511,6 +511,8 @@ def cmd(ctx,
         deploy_tasks = {}
         for index, (project, project_requires, deploy_target, deploy_options) in \
                 enumerate(sorted(projects_to_deploy, key=lambda x: x[0])):
+            if not deploy_options.get('enable'):
```
Comment (La0) [inline lib/please_cli/please_cli/decision_task.py]: @garbas The deployment will be disabled if the option is missing. I think we should either throw an exception if it's missing, or allow the deployment by default

**Comment by rv404674** [comment]
hey @La0 , i have added the case to throw an exception when enable field is missing.

File: lib/please_cli/please_cli/decision_task.py [hunk 2]
```diff
@@ -511,6 +511,13 @@ def cmd(ctx,
         deploy_tasks = {}
         for index, (project, project_requires, deploy_target, deploy_options) in \
                 enumerate(sorted(projects_to_deploy, key=lambda x: x[0])):
+            try:
+                enable = deploy_options['enable']
+            except KeyError:
+                raise click.ClickException('Missing `enable` in deploy options' )
```
Comment (garbas) [inline lib/please_cli/please_cli/decision_task.py]: :+1: can you also mention which project and channel this is enable option is missing at?

**Comment by rv404674** [comment]
hey @garbas , i have added your suggestions. I think it is complete now. Please can you take a look when you get some time. Thanks. :)

**Comment by garbas** [comment]
testing this PR locally

**Comment by garbas** [comment]
works like magic! thank you, thank you, thank you.

**Pull request merged**